From ecdc6c0d3c3e40843916427923a45a93d748c1c6 Mon Sep 17 00:00:00 2001
From: Andrei Pelinescu-Onciul <andrei@iptel.org>
Date: Thu, 12 Nov 2009 16:03:39 +0100
Subject: [PATCH 11/53] api: SCTP_DELAYED_SACK typo fix

According to the api draft, the socket option for setting the
delayed sack timer should be called SCTP_DELAYED_SACK and not
SCTP_DELAYED_ACK.
Added a new define for SCTP_DELAYED_SACK (SCTP_DELAYED_ACK is not
removed to maintain compatibility with existing applications).

Signed-off-by: Andrei Pelinescu-Onciul <andrei@iptel.org>
Signed-off-by: Vlad Yasevich <vladislav.yasevich@hp.com>
---
 src/include/netinet/sctp.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/include/netinet/sctp.h b/src/include/netinet/sctp.h
index 99606d1..888ce1b 100644
--- a/src/include/netinet/sctp.h
+++ b/src/include/netinet/sctp.h
@@ -98,6 +98,7 @@ enum sctp_optname {
 	SCTP_DELAYED_ACK,
 #define SCTP_DELAYED_ACK_TIME SCTP_DELAYED_ACK
 #define SCTP_DELAYED_ACK SCTP_DELAYED_ACK
+#define SCTP_DELAYED_SACK SCTP_DELAYED_ACK
 	SCTP_CONTEXT,	/* Receive Context */
 #define SCTP_CONTEXT SCTP_CONTEXT
 	SCTP_FRAGMENT_INTERLEAVE,
-- 
1.7.11.7

